### PR TITLE
Add Cortex-M0 SVD

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

File: CMSIS-SVD/Cortex_M/cm0.svd [hunk 1]
```diff
@@ -0,0 +1,928 @@
...
+            <field>
+              <name>NMIPENDSET</name>
+              <description>NMI set-pending bit.</description>
+              <bitOffset>31</bitOffset>
+              <bitWidth>1</bitWidth>
+              <access>read-write</access>
+            </field>
+          </fields>
+        </register>
+
```
Comment (lambourg) [inline CMSIS-SVD/Cortex_M/cm0.svd:231]: the VTOR register definition is missing

File: CMSIS-SVD/Cortex_M/cm0.svd [hunk 2]
```diff
@@ -0,0 +1,928 @@
...
+          <description>Interrupt Clear-Pending Registers</description>
+          <addressOffset>0x180</addressOffset>
+          <size>32</size>
+          <access>read-only</access>
+        </register>
+        <register>
+          <name>NVIC_IPR</name>
+          <description>Interrupt Priority Register</description>
+          <addressOffset>0x300</addressOffset>
+          <size>8</size>
```
Comment (lambourg) [inline CMSIS-SVD/Cortex_M/cm0.svd]: the IPR registers are only word-accessible, so should be binded to a 32-bit register.

**Review by lambourg — CHANGES_REQUESTED** [review]

File: CMSIS-SVD/Cortex_M/cm0.svd (hunk 2 above)
Comment (Fabien-Chouteau) [reply CMSIS-SVD/Cortex_M/cm0.svd]: Fixed.

File: CMSIS-SVD/Cortex_M/cm0.svd (hunk 1 above)
Comment (Fabien-Chouteau) [reply CMSIS-SVD/Cortex_M/cm0.svd:231]: Unlike any other Cortex-M, there's no VTOR on the M0.

Comment (lambourg) [reply CMSIS-SVD/Cortex_M/cm0.svd:231]: Well, the ARM documentation says it's optional.

http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0662a/Ciheijba.html

Comment (Fabien-Chouteau) [reply CMSIS-SVD/Cortex_M/cm0.svd:231]: That's the M0+.

**Review by lambourg — APPROVED** [review]

**Comment by simonjwright** [comment]
On 14 Feb 2017, at 11:32, Lambourg Jérôme <notifications@github.com> wrote:

> Well, the ARM documentation says it's optional.
> http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0662a/Ciheijba.html <http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0662a/Ciheijba.html>That's for the M0+; the M0 doesn't have a VTOR - http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0662a/Ciheijba.html

BTW, your link is to the superseded r0p0.

**Pull request merged**